locale/en/emails.po

```diff
@@ -518,15 +518,21 @@ msgstr ""
 "submission to your ORCID profile on publication.<br>\n"
 "Visit the link to the official ORCID website, login with your profile and "
 "authorize the access by following the instructions.<br>\n"
-"<a href=\"{$authorOrcidUrl}\"><img id=\"orcid-id-logo\" src=\""
+"<br>\n"
+"<a href=\"{$authorOrcidUrl}\" style=\"display: inline-flex; align-items: center; background-color: white; text-align: center; padding: 10px 20px; text-decoration: none; border-radius: 5px; border: 2px solid #d7d4d4;\">"
```

Comment:
This looks okay to me, but just want to run it by @Devika008 since (to my knowledge) this is the first "button" we've included in emails. Just want to make sure it's following best practices for compatibility and accessibility, especially given the varying degrees of CSS support in email clients.

Reply:
I've done a bit more looking into things and this should be okay.